_config.yml

```diff
@@ -96,6 +96,7 @@ publons_id: # your ID on Publons
 quora_username: # your Quora username
 research_gate_profile: # your profile on ResearchGate
 scholar_userid: qc6CJjYAAAAJ # your Google Scholar ID
+inspirehep_id: 1010907 # Inspire HEP author ID
```

Comment:
Here we used the fields in alphabetical order. Please add it accordingly.

Reply:
I apologize for not noticing it!